.github/workflows/gen-llms.yml

```
@@ -0,0 +1,69 @@
+jobs:
+  regenerate:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'pull_request' &&
+       vars.LLMS_MANUAL != 'true' &&
+       github.event.pull_request.head.repo.full_name == github.repository)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Resolve target branch
+        id: target
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)